tests/integration_tests/framework/hello_world.py

```diff
@@ -70,16 +71,15 @@ def _webserver_request():
         return deployment
 
     def _assert_hello_world_events(self, deployment_id):
-        rest_client = test_utils.create_rest_client()
-        events = rest_client.events.list(
+        events = self.test_case.client.events.list(
             deployment_id=deployment_id)
         self.test_case.assertGreater(len(events.items), 0)
 
     def _prepare_hello_world(self):
         logger = self.test_case.logger
         repo_name = 'cloudify-hello-world-example'
-        branch = self.test_case.env.core_branch_name
-        workdir = self.test_case.env.test_working_dir
+        branch = os.environ.get(BRANCH_NAME_CORE, 'master')
```

Comment:
oh nooo, it's the "random undocumented envvars" again, let's not do this

and actually, I don't see what does this test_hello_world test (which is split between 3 files - this, `agent_tests/test_hello_world.py`, and the base `test_cases.py`) actually test, that isn't tested by just the agent install test

drop it if you want, otherwise i'll drop it